apps/addons/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^search/$', views.SearchView.as_view(), name='addons.api.search'),
+]
```

Comment:
I was lazy and it was easier. I think Marketplace had `addons/api.py` files at one point and then the get pulled into the views. I'm not sold on this but it made setting up the URLs easier so it should work for running a local search API.